### PR TITLE
Add Katello 3.4 documentation

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: plugins/katello/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: plugin_index
 title: Katello Documentation
-versions: [3.3, 3.2, 3.1, 3.0, 2.4, nightly]
+versions: [{name: 3.4, display: '3,4-RC'}, {name: 3.3, display: '3.3'}, {name: 3.2, 'display': '3.2'}, {name: 3.1, display: '3.1'}, {name: 3.0, display: '3.1'}, {name: 2.4, display: '2.4'}, {name: 'nightly', display: 'nightly'}]
```
Comment (jlsherrill) [inline plugins/katello/index.md]: `{name: 3.0, display: '3.1'}`  is not right

File: _layouts/plugins/katello/documentation.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 pagename: Plugin Manuals
-versions: [3.3, 3.2, 3.1, 3.0, 2.4, nightly]
+versions: [{name: 3.4, display: '3.4-RC'}, {name: 3.3, display: '3.3'}, {name: 3.2, 'display': '3.2'}, {name: 3.1, display: '3.1'}, {name: 3.0, display: '3.1'}, {name: 2.4, display: '2.4'}, {name: 'nightly', display: 'nightly'}]
```
Comment (jlsherrill) [inline _layouts/plugins/katello/documentation.html]: same issue here

**Comment by ehelms** [comment]
Updated @jlsherrill

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**